src/core/File.ts

```diff
@@ -451,6 +451,9 @@ export class File {
                 this.context.fuse.producer.addWarning('unresolved',
                     `Statement "${this.info.fuseBoxPath}" has failed to resolve in module "${this.collection && this.collection.name}"`);
             }
+            if (/\.html$/.test(this.info.fuseBoxPath)){
```

Comment:
I don't think we need to need a special case for HTML. 
IT should be generic and message customised.